### ISSUE TITLE
Keyboard selection is broken in projects area in package manager for VS

### ISSUE THREAD

**Issue by zihotki** [NuGet/Home#5178]
## Details about Problem

NuGet product used (NuGet.exe | VS UI | Package Manager Console | dotnet.exe): VS UI

NuGet version (x.x.x.xxx): 4.1.0

VS version (if appropriate): 2017 Enterprise v. 15.1

OS version (i.e. win10 v1607 (14393.321)): win10 

Worked before? If so, with which NuGet version:
As far as I remember it worked in some 3.* or 2.* version in VS 2015 SP3

## Detailed repro steps so we can see the same problem

1. Open solution with several projects and open NuGet package manager
2. Find any nuget package, select it and click inside the area with projects

### What doesn't work there:
1. Using Up and Down arrows you can't navigate the projects but by mouse clicking you can do that as well as Page Up and Page Down
2. Shift+Up/Down doesn't work but using Shift + Mouse Click you can do that as well as Shift + PageUp and Shift + PageDown
3. After you selected several projects Space toggles only first project instead toggling the whole selection

**Comment by mishra14** [comment]
Verified as not working. @alpaix can you please drive this accessibility issue?

**Comment by mishra14** [comment]
assigning to @PatoBeltran  who now owns accessibility tenet.

**Comment by mishra14** [comment]
related: https://github.com/NuGet/Home/issues/5207
